state/azure/cosmosdb/cosmosdb.go

```diff
@@ -15,17 +15,18 @@ package cosmosdb
 
 import (
 	// For go:embed.
```

Comment:
Delete this comment and remove the `_ "embed"` import as those are not necessary anymore. Also, you should delete everything in the `storedprocedures` folder